index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+  <body>
+    <header class="container">
+      <nav class="top-bar">
+        <div class="status">
+          <div id="time-container">
+            <img
+              id="time-icon"
+              src="images/Time Style.png"
+              alt="not available!"
+              class="time"
+            />
+          </div>
+          <div id="status-container">
+            <img
+              id="status-icon"
+              src="images/Group 3.png"
+              alt="not available!"
+              class="internet-wifi-charge"
+            />
+          </div>
+        </div>
+        <div class="header">
+          <div class="text-welcome">
+            <p class="welcome">
+              <a href="#">Welcome</a>
+            </p>
+          </div>
+          <nav class="humberger-icon-container">
+            <img
+              id="humberger-icon"
+              src="images/Enabled.png"
+              alt="not available"
+            />
+          </nav>
+        </div>
+      </nav>
+      <section class="headline">
+        <div class="primary-text">
+          <span>
+            <h1>Hey there. I'm Bakhteyari</h1>
+          </span>
+          <span id="colored-text">
+            <h1>I'm a software developer</h1>
+          </span>
```

Comment:
- [ ] **[OPTIONAL]** Kindly use the `h1` tag once in your project for the title, for the subtitle we could use the `h2` tag, it is for a convention, semantics, and user experience(UX). 👍🏽